main.tf

```diff
@@ -1,8 +1,8 @@
 locals {
   enabled = module.this.enabled
 
-  tags = merge(
-    var.tags,
+  node_group_tags = merge(
```

Comment:
I modified this because the cluster-level tags were being applied to all resources in the module (e.g. IAM roles, policies, etc.), and we probably don't want them there.